Resources/Private/Fusion/Content/Columns/Four.fusion

```diff
@@ -0,0 +1,30 @@
+##
+# "FourColumn" element
+#
+prototype(Neos.Demo:Content.Columns.Four) < prototype(Neos.Neos:ContentComponent) {
+
+    column0 = Neos.Neos:PrimaryContent {
```

Comment:
What's the exact reason to use `PrimaryContent` and not just `ContentCollection` here?
I always thought there should just be one `PrimaryContent` in the page rendering.
Some plugins use this to replace one part of the rendering with their own output.

Reply:
@Sebobo It was just bad habit and is adjusted now